examples/images/diffusion/ldm/util.py

```diff
@@ -45,7 +39,7 @@ def ismap(x):
 
 
 def isimage(x):
-    if not isinstance(x, torch.Tensor):
+    if not isinstance(x,torch.Tensor):
         return False
```

Comment:
Did you use `pre-commit`? This file seems to contain mainly format change. `pre-commit` should format the code in a way that there is a space between `x` and `torch.Tensor`.